thirdparty/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+include(FetchContent)
+
+###################################################################################################
+# - cub -------------------------------------------------------------------------------------------
+
+FetchContent_Declare(
+    cub
+    GIT_REPOSITORY https://github.com/thrust/cub.git
+    GIT_TAG        1.9.10
+    GIT_SHALLOW    true
+    PATCH_COMMAND
+        COMMAND    sed -i "508,509 s/$<SEMICOLON>/ = 0$<SEMICOLON>/" cub/device/dispatch/dispatch_histogram.cuh
```

Comment:
Is this something we could patch upstream in cub so that when there's a 1.9.11 for example we could use it without the need for patching?

Reply:
Yeah, can you file a PR?

Reply:
https://github.com/thrust/cub/pull/35